.woodpecker.yml

```diff
@@ -197,7 +205,12 @@ steps:
       PGHOST: database
       PGDATABASE: lemmy
     commands:
-      - cargo install diesel_cli
+      # Install cargo binstall
+      - wget https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz
+      - tar -xvf cargo-binstall-x86_64-unknown-linux-musl.tgz
+      - cp cargo-binstall /usr/local/cargo/bin
```

Comment:
Wget and tar should not need to be run again. It might be possible to also remove cp and replace `cargo binstall` with `./cargo-binstall`.